core/db/session.go

```diff
@@ -30,6 +30,7 @@ type Session struct {
 	MeterStart      *float64  `json:"meterStart" csv:"Meter Start (kWh)" gorm:"column:meter_start_kwh"`
 	MeterStop       *float64  `json:"meterStop" csv:"Meter Stop (kWh)" gorm:"column:meter_end_kwh"`
 	ChargedEnergy   float64   `json:"chargedEnergy" csv:"Charged Energy (kWh)" gorm:"column:charged_kwh"`
+	ChargeDuration  *float64  `json:"chargeDuration" csv:"Charge Duration (s)" gorm:"column:charge_duration_s"`
```

Comment:
Warum keine `time.Duration` einschl. Formatierung in `(t *Sessions) writeRow`?

Reply:
Das führt dazu, dass wir den Wert in `ns` in der Datenbank abspeichern und auch im JSON so bekommen. Ich find das etwas unhandlich. Die Änderung in https://github.com/evcc-io/evcc/pull/9309 macht zwar, dass das im CSV dann menschenlesbar formatiert ist (12m22s), will man die Daten aber weiterverarbeiten muss man wieder zurückparsen. Numbers kann das scheinbar, aber ich hab das Gefühl die Änderung machts eher komplizierter als einfacher.

Reply:
Die Verarbeitung von time.Duration ist ab Werk enthalten. Sekunden hätten nur einen Vorteil wenn ich die direkt aus der DB verarbeiten wollte?